minjur.cpp

```diff
@@ -69,13 +70,14 @@ class JSONHandler : public osmium::handler::Handler {
 
 public:
 
-    JSONHandler(bool create_polygons, const tileset_type& tiles, unsigned int zoom, const std::string& error_file) :
+    JSONHandler(bool create_polygons, const tileset_type& tiles, unsigned int zoom, const std::string& error_file, const std::string& attr_prefix) :
         m_buffer(),
         m_geometry_error_count(0),
         m_create_polygons(create_polygons),
         m_tiles(tiles),
         m_zoom(zoom),
         m_error_stream(nullptr),
+        m_attr_prefix(attr_prefix),
```

Comment:
Ah, here the initialization is done in the right section. But there is another problem here: The class members should always be initialized in the order they are declared. So `m_attr_prefix` has to come after `m_filter`. The compiler should have warned you about that. The reason is that C++ will always initialize the members in the order they are declared and not the order those initialization lines are in. That can be quite confusing, so it is better to always keep those in the same order.